dist/origin-web-common-services.js

```diff
@@ -1218,13 +1218,17 @@ angular.module("openshiftCommonServices")
     };
 
     var isServiceBindable = function(serviceInstance, serviceClasses) {
-      var serviceClass = getServiceClassForInstance(serviceInstance, serviceClasses);
-
       // If being deleted, it is not bindable
       if (_.get(serviceInstance, 'metadata.deletionTimestamp')) {
         return false;
       }
 
+      // If provisioning failed, the service is not bindable
+      if ($filter('isBindingFailed')(serviceInstance, 'Failed')) {
```

Comment:
Do we want an `isServiceInstanceFailed` filter? Even if they're the same implementation, it might be good to have separate filters so the names match. Then we can always change the logic for one later if we need to. We do this for `isServiceInstanceReady` and `isBindingReady`

Opinion?

Reply:
Sounds reasonable.

Reply:
Switch this to use the new `serviceInstanceFailed` filter since it's an instance?

Reply:
Fixed. :)